src/commands/object/create.js

```diff
@@ -40,6 +40,14 @@ class CreateCommand extends BaseCommand {
       data = await this._readStdin();
     }
 
+    if (typeof data === 'string') {
+      try {
+        data = JSON.parse(data);
+      } catch (err) {
+        console.error(`Unable to parse due to: ${err}`);
```

Comment:
This could be expected. Remove. `data` can be just a string.

Reply:
If data can be an arbitrary string, what is expected then? This seems like an extremely easy case for mistakes and failures to make its way into the system as there would be no validation?

Reply:
More importantly, what happens when an arbitary string is uploaded? What use case is expected here?